File: DesktopApp/Praktikumsverwaltung_DesktopApp/Praktikumsverwaltung_DesktopApp/pkgData/Pupil.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Praktikumsverwaltung_DesktopApp.pkgData
{
    class Pupil : Person
    {
        public string IdDepartment { get; set; }
        public string IdClass { get; set; }                

        public Pupil(string id, string firstname, string lastname, string email, string username, string password, bool isActive, string idDepartment, string idClass)
        : base(id, firstname, lastname, email, username, password, isActive)
        {
            IdDepartment = idDepartment;
            IdClass = idClass;
        }
    }
}

File: PraktikumsverwaltungWebService/src/java/pkgData/Pupil.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pkgData;

import javax.xml.bind.annotation.XmlRootElement;
import org.bson.types.ObjectId;

/**
 *
 * @author schueler
 */
@XmlRootElement
public class Pupil extends Person{
    private ObjectId idDepartment;
    private ObjectId idClass;

    public Pupil() {
        
    }
    
    public Pupil (String username, String password) {
        super(username, password);
    }

    public Pupil(ObjectId id, String username, String password, String firstName, String lastname, String currentForm, String email, ObjectId idDepartment, ObjectId idClass, boolean isActive) {
        super(id, username, password, firstName, lastname, email, isActive);
        this.idDepartment = idDepartment;
        this.idClass = idClass;
    }

    public ObjectId getIdDepartment() {
        return idDepartment;
    }

    public void setIdDepartment(ObjectId idDepartment) {
        this.idDepartment = idDepartment;
    }

    public ObjectId getIdClass() {
        return idClass;
    }

    public void setIdClass(ObjectId idClass) {
        this.idClass = idClass;
    }
    
    
}